app/controllers/users_controller.rb

```diff
@@ -62,9 +62,36 @@ def filters
     end
   end
 
+  def make_owner_page
```

Comment:
whats the use of this route?

Reply:
This will take you to the page where you will select the person who you want to make owner.